portal-frontend/src/styles/typography.scss

```diff
@@ -0,0 +1,65 @@
+@import '@bcgov/bc-sans/css/BCSans.css';
```

Comment:
Could you add comment somewhere that we need to use relative values in typography

Reply:
Done! Added a comment above the first h1